src/Hydration/PublicPropertyManager.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Livewire\Hydration;
+
+use Livewire\Exceptions\CannotRegisterPublicPropertyWithoutImplementingWireableException;
+use Livewire\Wireable;
+
+class PublicPropertyManager
```

Comment:
Can you move this to the base Livewire directory `src` and rename to `LivewirePropertyManager`?

Reply:
This seems to be outdated as well, but I'll take a look.

Reply:
Github was showing old code. Has refreshed now so this comment is irrelevant.